deprecated/components/FileUploader/index.stories.js

```diff
@@ -4,7 +4,7 @@ import { boolean } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
 
 import Heading from '../Heading';
-import FileUploader from '.';
```

Comment:
Why change all these if they are now deprecated?

Reply:
This was done automatically by IntelliJ when I moved the files